worlds/alttp/__init__.py

```diff
@@ -819,20 +787,17 @@ def fill_slot_data(self):
             # for convenient auto-tracking of the generated settings and adjusting the tracker accordingly
 
             slot_options = ["crystals_needed_for_gt", "crystals_needed_for_ganon", "open_pyramid",
```

Comment:
may i request to add "triforce_pieces_required" here to the list? since i seem to missed that to add originally for the tracker

Reply:
Went ahead and added all the triforce piece options